### ISSUE TITLE
Popover - wrong import statement

### ISSUE THREAD

**Issue by TalZa1810** [wix/wix-style-react#3983]
# Documentation Issue

### 🔦 Scope

`Popover` documentation

Relevant Page URL:  https://wix-wix-style-react.surge.sh/?selectedKind=Components&selectedStory=Popover&full=0&addons=0&stories=1&panelRight=0
### 💁 Explanation

The `import` statement is 
```
import undefined from 'wix-style-react/undefined';
```

instead of:
```
import Popover from 'wix-style-react/Popover';
```

### 👀 Severity

- [ ] Major
- [x] Low

**Comment by shlomitc** [comment]
strange. @argshook - everything looks ok from a quick look. any idea?

**Comment by argshook** [comment]
found the issue, there's a bug in react-autodocs-utils:

during component parsing it goes to this line (https://github.com/wix/wix-style-react/blob/master/src/Popover/Popover.js#L10) and goes into export, missing the popover class

i'll fix this on https://github.com/wix/react-autodocs-utils so no
changes will be needed on wix-style-react part

**Comment by argshook** [comment]
i released react-autodocs-utils 3.5.2 with a fix, can confirm it fixes Popover docs page. wsr locally looks ok, next release will also have a fix on surge.sh

closing, please reopen if it happens again
